include/llbuild/Basic/ShellUtility.h

```diff
@@ -13,8 +13,11 @@
 #ifndef LLBUILD_BASIC_SHELLUTILITY_H
 #define LLBUILD_BASIC_SHELLUTILITY_H
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wshorten-64-to-32"
```

Comment:
I would rather not do this, it is too hard to maintain. I don't find this warning really adds enough value to be worth the impact on the code, I would rather turn it off TBH.